TypoSearch.py

```diff
@@ -115,29 +112,30 @@ def main():
                         required=True)
     parser.add_argument("-table", action="store", dest="table", help="Name of table to search in", required=True)
     parser.add_argument("-rank", action="store", dest="rank", help="Rank level to start searching at", required=True)
```

Comment:
is rank an integer or string? Help message could be more helpful.

Reply:
`-rank` is an integer, help message has been updated.